etc/kayobe/inventory/group_vars/overcloud/cis

```diff
@@ -159,4 +163,20 @@ ubtu22cis_max_log_file_size: 1024
 # ubtu22cis_bootloader_password_hash
 ubtu22cis_rule_1_4_1: false
 ubtu22cis_rule_1_4_3: false
+
+# Disable: Ensure minimum days between password changes is configured
+ubtu22cis_rule_5_5_1_1: false
+
+# Disable: Ensure password expiration is 365 days or less
+ubtu22cis_rule_5_5_1_2: false
+
+# Disable: Ensure password expiration warning days is 7 or more
```

Comment:
Do we need to disable the password expiration warning is 7 days or more? I think it should be enabled if anything?

Reply:
Disabling the rule just means that we won't make any changes to the current policy, but you are right, this one seems relatively benign. This would have the effect of changing the default to 7 days for new users (which is also the distro default, so wouldn't have an effect on new systems).

Reply:
done - thanks